README.md

```diff
@@ -253,7 +253,7 @@ The supported rules are described below:
 ##### operations
 | Rule                         | Description                                                                         | Spec     |
 | ---------------------------- | ----------------------------------------------------------------------------------- | -------- |
-| undefined_tag                | Flag a tag that is in operations and not listed in `tags` on the top level.         | shared   |
+| undefined_tag                | Flag a tag that no_parameteris in operations and not listed in `tags` on the top level.         | shared   |
```

Comment:
Is this an intentional change?

Reply:
great catch... that must have been the result of an errant "paste" of "no_parameter" :)